services/api/service.go

```diff
@@ -789,7 +789,7 @@ func (api *RelayAPI) handleGetPayload(w http.ResponseWriter, req *http.Request)
 
 		err = api.db.SaveDeliveredPayload(bidTrace, payload)
 		if err != nil {
-			log.WithError(err).Error("failed to save delivered payload")
+			log.WithError(err).WithField("bidTrace", bidTrace).Error("failed to save delivered payload")
```

Comment:
that field (signed blinded beacon block) could be ignored, it's never exported or used anywhere, more stored for reference and completeness of data.

although i guess it would be reasonable to log this field too on such an error 🤔

Reply:
added